.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "python.pythonPath": "C:\\Users\\Ethan\\.virtualenvs\\cs160-j7hg6mN6\\Scripts\\python.exe"
+    "python.pythonPath": "C:\\Users\\ddoem\\.virtualenvs\\projectdev-TXkBElKe\\Scripts\\python.exe"
 }
```

Comment:
Fix new line

Reply:
This was added to `.gitignore` but not added to the repo. @ethanmnrd should use `git rm --cached .vscode/settings.json` if you don't want it to be committed to the repo anymore.